index.html

```diff
@@ -41,11 +41,11 @@
 
     // $eventLog.innerHTML += '\n\n';
     simplepicker.on('submit', (date, readableDate) => {
-      $eventLog.innerHTML += readableDate + '\n';
+      $evnetLog.innerHtml += `${readableDate} \n`
     });
 
     simplepicker.on('close', (date) => {
-      $eventLog.innerHTML += 'Picker Closed'  + '\n';
+      $evnetLog.innerHtml += `Picker Closed \n`
```

Comment:
The variable name change here `evnetLog` from `eventLog` is wrong. It should be `eventLog.innerHTML` in both places.